src/lib/utils.ts

```diff
@@ -34,3 +37,47 @@ export const loadScriptTag = async (srcValue: string): Promise<void> => {
 
   await loadedPromise
 }
+
+export const getUserVariables = (): {userId: string, courseId: string, instance: string} | null => {
+  const userId = '2'
```

Comment:
We don't use / need `userId` right now so I'd remove it to avoid confusion. 

Also, what do you think about calling this `getUserContext` or `getCurrentContext`?